tests/gtest/aviftest_helpers.h

```diff
@@ -13,6 +13,17 @@ namespace libavif
 namespace testutil
 {
 
+enum avifChannel
```

Comment:
I believe C++ allows us to use just `R`, `G`, `B`, `A`. We need to define this as an enum class to force the use of `avifChannel::R`, `avifChannel::G`, `avifChannel::B`, `avifChannel::A`.